schema/sale_artwork.js

```diff
@@ -40,10 +39,15 @@ const SaleArtworkType = new GraphQLObjectType({
       },
       bid_increments: {
         type: new GraphQLList(GraphQLFloat),
-        resolve: ({ minimum_next_bid_cents, sale_id }) => {
-          return gravity(`sale/${sale_id}`).then(sale => {
-            return gravity("increments", {
-              key: sale.increment_strategy,
+        resolve: (
+          { minimum_next_bid_cents, sale_id },
+          options,
+          request,
+          { rootValue: { incrementsLoader, saleLoader } }
+        ) => {
+          return saleLoader(sale_id).then(sale => {
+            return incrementsLoader({
+              key: sale.incremenet_strategy,
```

Comment:
This doesn’t appear to be spelled correctly.

Reply:
So there must also be a missing/faulty test.

Reply:
Yeah I'm kind of unsure how this didn't fail, lots of tests around increments. Let me look into it